README.md

```diff
@@ -144,7 +144,6 @@ Flags:
 | Argument         | Description                                                      | Default Value |
 |------------------|------------------------------------------------------------------|:-------------:|
 | **--chown**      | chown the ouput directory to match the specified UID:GID         |       ❌      |
-| **--config**     | Path to a [build config](#-build-config)                         |       ❌      |
```

Comment:
The thing is this is used elsewhere so I think we may be partially committed short term.

(There is some things related to this, like the fact I don't quite understand why it's JSON and not TOML as seems would be expected for blueprints, but that's an aside)

Reply:
> I don't quite understand why it's JSON and not TOML

No real (good) reason other than the initial version BIB was heavily based on https://github.com/osbuild/images/blob/main/cmd/build/main.go, which is used for test image builds and it was simpler to use the json formatting for config files.

Since we're moving towards making it more explicit that this config *is* an instance of an Image Builder blueprint, I agree that we should make it a toml to make the connection clearer.